src/sbl_filing_api/routers/filing.py

```diff
@@ -125,7 +125,7 @@ async def upload_file(
     submission.state = SubmissionState.SUBMISSION_UPLOADED
     submission = await repo.update_submission(submission)
     background_tasks.add_task(
-        submission_processor.validate_and_update_submission, period_code, lei, submission, content
+    background_tasks.add_task(submission_processor.validate_and_update_submission, lei, submission, content)
```

Comment:
not sure why period_code was removed here but it is needed for the upload_to_storage function.